app/src/main/res/layout/time_dialog.xml

```diff
@@ -64,6 +65,45 @@
                 android:layout_height="wrap_content"
                 android:id="@+id/disableSleeptimerButton"/>
 
+        <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal">
+
+            <Button
+                android:id="@+id/extendSleepFiveMinutesButton"
+                style="?attr/materialButtonOutlinedStyle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="4dp"
+                android:layout_marginRight="4dp"
+                android:layout_weight="1"
+                android:padding="5dp"
+                android:text="+5 min" />
```

Comment:
Please use `tools:text` here to make it clear that it is only used for the layout preview and that the actual text is set using Java.